README.md

```diff
@@ -41,8 +41,10 @@ import { routerForBrowser } from 'redux-little-router';
 import yourReducer from './your-app';
 
 // Define your routes in a route-to-anything hash like below.
-// The value of the route key can be any serializable data.
-// This data gets attached to the `router` key of the state
+// The value of the route key can be any serializable data,
+// including an empty object.
```

Comment:
The example below has all routes with assigned titles, but empty objects are fine and that's nice to know.